aldryn_forms/models.py

```diff
@@ -207,6 +210,26 @@ class EmailFieldPlugin(FieldPluginBase):
     )
 
 
+class FileUploadFieldPlugin(FieldPluginBase):
+    upload_to = FilerFolderField(
+        verbose_name=_('Upload files to'),
+        help_text=_('Select a folder to which all files submitted through '
+                    'this field will be uploaded to.')
+    )
+    max_size = FileSizeField(
+        verbose_name=_('Maximum file size'),
+        null=True, blank=True,
+        help_text=_('The maximum file size of the upload, in bytes. You can '
+                    'use common size suffixes (kB, MB, GB, ...).')
+    )
+
+
+FileUploadFieldPlugin._meta.get_field('help_text').help_text = _(
```

Comment:
You can't override a previously defined field. As @czpython suggested, here is a proposal to do so in the form: https://github.com/aldryn/aldryn-forms/commit/494e7190c475d65f9ad1043922cfaac841402a5a